packages/app/src/domain/vaccine/vaccine-coverage-toggle-tile.tsx

```diff
@@ -1,69 +1,57 @@
-import { TwoKpiSection } from '~/components/two-kpi-section';
-import { KpiTile } from '~/components/kpi-tile';
-import { RadioGroup } from '~/components/radio-group';
+import { css } from '@styled-system/css';
 import { useState } from 'react';
+import { isPresent } from 'ts-is-present';
 import { Box } from '~/components/base';
-import { css } from '@styled-system/css';
+import { KpiTile } from '~/components/kpi-tile';
 import { KpiValue } from '~/components/kpi-value';
 import { Markdown } from '~/components/markdown';
-import { replaceVariablesInText } from '~/utils/replace-variables-in-text';
+import { RadioGroup } from '~/components/radio-group';
+import { TwoKpiSection } from '~/components/two-kpi-section';
 import { InlineText } from '~/components/typography';
 import { parseBirthyearRange } from '~/domain/vaccine/logic/parse-birthyear-range';
+import { parseFullyVaccinatedPercentageLabel } from '~/domain/vaccine/logic/parse-fully-vaccinated-percentage-label';
 import { useIntl } from '~/intl';
 import { assert } from '~/utils/assert';
+import { replaceVariablesInText } from '~/utils/replace-variables-in-text';
 
-type AgeGroupTypes = {
-  description_vaccination_grade: string;
-  description_vaccination_one_shot: string;
-  label: string;
+type ageTypes = {
+  fully_vaccinated: number | null;
+  has_one_shot: number | null;
+  birthyear: string;
+  label_fully_vaccinated: string | null;
+  label_has_one_shot: string | null;
```

Comment:
```suggestion
type AgeTypes = {
  fully_vaccinated: number | null;
  has_one_shot: number | null;
  birthyear: string;
  label_fully_vaccinated: string | null;
  label_has_one_shot: string | null;
```
Also, doesn't this type get generated? Or is this for mock purposes?

Reply:
We have 2 completely different schemas that need to fit together in 1 component, it was changed last minute where the data should come from. I couldn't find a better solution.